start.sh

```diff
@@ -40,6 +47,6 @@ else
   if [[ ! -z "$SPARK_GID" && "$SPARK_GID" != "$(id -g)" ]]; then
     echo 'Container must be run as root to set $SPARK_GID'
   fi
-  echo "Execute the command"
-  exec $*
+  echo "Need to be run as root"
```

Comment:
Could you make this message more descriptive telling the user how to do this.
`This container needs to be run as the root user to allow configuration prior to starting Spark. Use the "--user root" flag for Docker or the securityContext: .... annotation for Kubernetes`